man/nanotime.Rd

```diff
@@ -85,6 +88,9 @@ A nanotime object
 Functions to operate on nanosecond time resolution using integer64
 bit representation. Convertions functions for several standard R
```

Comment:
Good catches, both typos look old.  I should check the homepage.

Also made hyphenation more consistent.